frontend/src/main/scala/bloop/cli/Proxy.scala

```diff
@@ -0,0 +1,90 @@
+package bloop.cli
```

Comment:
Can we move it to the `bloop.util` package?

Reply:
Yes of course